### PR TITLE
DartIntegration-v3

### PR DESCRIPTION
**Pull request by costashatz**
**DartIntegration:** This pull request fixes a small bug in the cylinder shape and adds support for the cone shape..

### CODE REVIEW

File: src/Magnum/DartIntegration/ConvertShapeNode.cpp [hunk 1]
```diff
@@ -121,6 +122,31 @@ Containers::Optional<ShapeData> convertShapeNode(dart::dynamics::ShapeNode& shap
 
             shapeData.meshes = Containers::Array<Trade::MeshData3D>(Containers::NoInit, 1);
             new(&shapeData.meshes[0]) Trade::MeshData3D{Primitives::capsule3DSolid(32, 32, 32, halfLength)};
+
+            Matrix4 rot = Matrix4::rotationX(Math::Rad<Float>(Math::Constants<Float>::piHalf()));
```
Comment (mosra) [inline src/Magnum/DartIntegration/ConvertShapeNode.cpp]: `Rad(Constants::piHalf())` ... there are convenience typedefs for this :) Same in the two other occurences below.

File: src/Magnum/DartIntegration/Test/ConvertShapeNodeTest.cpp [hunk 2]
```diff
@@ -436,8 +436,18 @@ void ConvertShapeNodeTest::urdf() {
                 CORRADE_COMPARE(shapeDataAll->materials[0].diffuseColor(), (Vector3{0.6f, 0.6f, 0.6f}));
                 CORRADE_COMPARE(shapeDataMaterial->materials[0].diffuseColor(), (Vector3{0.6f, 0.6f, 0.6f}));
             } else {
+                #if DART_MAJOR_VERSION > 6 || ((DART_MAJOR_VERSION == 6) && (DART_MINOR_VERSION >=9))
+                if(shapeNode->getName() == "base_link_ShapeNode_0") {
+                    CORRADE_COMPARE(shapeDataAll->materials[0].diffuseColor(), 0x0_srgbf);
```
Comment (mosra) [inline src/Magnum/DartIntegration/Test/ConvertShapeNodeTest.cpp]: `0x000000_srgbf` pretty please :) I know it's the same but I'd like to follow how CSS colors look. Same below.

File: src/Magnum/DartIntegration/ConvertShapeNode.cpp (hunk 1 above)
Comment (costashatz) [reply src/Magnum/DartIntegration/ConvertShapeNode.cpp]: Should be fixed now!

File: src/Magnum/DartIntegration/Test/ConvertShapeNodeTest.cpp (hunk 2 above)
Comment (costashatz) [reply src/Magnum/DartIntegration/Test/ConvertShapeNodeTest.cpp]: OK :)

**Comment by mosra** [comment]
Merged in e78af90b475f98d992a3493c33e4d85a3f592bcf, 398feb7dd2489d4baeb62e200a56bb0471f2553f, 50e7ec2f1dba6e282a219301ba7ba60280462b63, 33f45c778d55761104fcd03fe26ec33b46083f42, 7964c6dc8505b1c1de2fa9c0860744304ad01113 and dc029da8c2fcba503442af90e54d3145aef4e166. Thanks a lot for the continued maintenance!

**Pull request closed without merging**